lms/djangoapps/learner_dashboard/views.py

```diff
@@ -73,4 +74,7 @@ def program_details(request, program_uuid):
         'user_preferences': get_user_preferences(request.user)
     }
 
-    return render_to_response('learner_dashboard/program_details.html', context)
+    if waffle.switch_is_active('new_program_progress'):
+        return render_to_response('learner_dashboard/new_program_details.html', context)
```

Comment:
I'm not a fan of using the word 'new' in files because it's confusing. Could we call it something like ```program_details_2017``` and rename the other one ```program_details_2016``` just so it's clearer? Or v1 and v2. I know after launch we'll clean it up but I think it's better to be clear until we get to that point.